### PR TITLE
[Integration][Gitlab] Add onboarding default resources

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What - 
- Added configurable batch size, and modified the project resync batch size
- Service Blueprint
- Production Readiness Scorecard
- 3 Actions
  - Scaffold Service
  - Send scorecard reminder
  - Enrich service
- 2 Pages
  - Production Readiness
  - Service Dashboard 

Screenshot 2023-12-31 at 16 59 53

Screenshot 2023-12-31 at 17 02 09

Screenshot 2023-12-31 at 17 02 41

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

**Comment by Tankilevitch** [comment]
Relies on #308

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
I have a feeling questions lets talk about it

**Review by yairsimantov20 — APPROVED** [review]
🌊 verbose changelog

File: integrations/gitlab/gitlab_integration/ocean.py [hunk 1]
```diff
@@ -69,14 +71,27 @@ async def on_resync(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
...
+        async for projects in service.get_all_projects():
+            # resync small batches of projects, so data will appear asap to the user.
+            # projects takes more time than other resources as it has extra enrichment performed for each entity
+            # such as languages, `file://` and `search://`
+            projects_batch_iter = iter(projects)
+            projects_processed_in_full_batch = 0
+            while projects_batch := tuple(
+                islice(projects_batch_iter, PROJECT_RESYNC_BATCH_SIZE)
+            ):
+                projects_processed_in_full_batch += len(projects_batch)
```
Comment (yairsimantov20) [inline integrations/gitlab/gitlab_integration/ocean.py:83]:
```
for index in range(0, len(projects), PROJECT_RESYNC_BATCH_SIZE):
 #  projects_processed_in_full_batch = i  + PROJECT_RESYNC_BATCH_SIZE + 1
  projects_batch = projects[i : i + PROJECT_RESYNC_BATCH_SIZE]

**Review by yairsimantov20 — APPROVED** [review]
🌊

**Pull request merged**